en/flight_controller/mindpx.md

```diff
@@ -79,18 +79,33 @@ make airmind_mindpx-v2_default
 
 ### Companion PC connection
 
-MindPX has a USB-TO-UART Bridge IC on the board.  A micro-USB to USB type A cable is used for the connection. Connect micro-USB end to the 'OBC' port of MindPX and USB type A end to companion computer.
+MindPX has a USB-TO-UART Bridge IC on the board.
+A micro-USB to USB type A cable is used for the connection. Connect micro-USB end to the 'OBC' port of MindPX and USB type A end to companion computer.
 
 And the max BAUD rate is the same with px4 family, which is up to 921600.
 
-### User Guide
+## User Guide
 
 > **Note** The user guide is [here](http://mindpx.net/assets/accessories/UserGuide9.18_2_pdf.pdf).
 
-## Where to buy
+## Where to Buy
 
 MindRacer is available at [AirMind Store](http://drupal.xitronet.com/?q=catalog) on internet. You can also find MindRacer at Amazon<sup>&reg;</sup> or eBay<sup>&reg;</sup>.
 
+## Serial Port Mapping
+
+UART | Device | Port
+--- | --- | ---
+USART1 | /dev/ttyS0 | RC
```

Comment:
This is from https://github.com/PX4/Firmware/blob/master/boards/airmind/mindpx-v2/src/board_config.h#L195 - not sure what "type" of RC input this can be used for.

Also not clear what USART6 and UART8 are mapped to.

Reply:
@iZhangHui can you please help fill this in?

Reply:
@iZhangHui can you please help fill this in?